test_app/tests/conftest.py

```diff
@@ -464,17 +465,15 @@ def mocked_gateway_view_get_request(self, *args, **kwargs):
 
 @pytest.fixture
 def system_user(db, settings, no_log_messages):
-    from test_app.models import User
```

Comment:
@john-westcott-iv I'm guessing this pattern was from some prior integration with testing in other repos that isn't needed because the testing utils exist now.